src/main.cpp

```diff
@@ -339,6 +339,63 @@ auto init_irods_connection_pool(const json& _config) -> irods::connection_pool
     };
 } // init_irods_connection_pool
 
+auto load_oidc_configuration(const json& _config) -> void
+{
+  nlohmann::json oi_config;
+        nlohmann::json endpoint_config;
+
+        // Scope as we're likely going to move this part out into a sep func...
+        try {
+			// Setup config
+            oi_config = _config.at(json::json_pointer{"/authentication/oidc"});
+			irods::http::globals::set_oidc_configuration(oi_config);
+
+            // Consider reusing context further down main?
+            net::io_context io_ctx;
+            net::ip::tcp::resolver tcp_res{io_ctx};
+            beast::tcp_stream tcp_stream{io_ctx};
+
+            // Load config
+            const auto host{oi_config.at("config_host").get_ref<const std::string&>()};
+            const auto port{std::to_string(oi_config.at("port").get<int>())};
+            const auto temp_uri{oi_config.at("well_known_uri").get_ref<const std::string&>()};
+			const auto uri{fmt::format("{}/.well-known/openid-configuration", temp_uri)};
```

Comment:
yeah, need that sweet-sweet clang-format.

Reply:
Deferring formatting discussions to this thread(?): https://github.com/irods/irods_client_http_api/pull/75#discussion_r1334508464

Reply:
Deferring formatting discussions to this thread(?): https://github.com/irods/irods_client_http_api/pull/75#discussion_r1334508464